pyccel/commands/pyccel_clean.py

```diff
@@ -56,32 +61,59 @@ def pyccel_clean(path_dir = None, recursive = True, remove_shared_libs = False,
         elif remove_programs and os.access(file_name, os.X_OK):
             os.remove(file_name)
 
-def pyccel_clean_command():
+def setup_pyccel_clean_parser(parser):
     """
-    Command line wrapper around the pyccel_clean function.
+    Add the pyccel-clean arguments to the parser.
 
-    A wrapper around the pyccel_clean function which allows
-    command line arguments to be passed to the function.
-    """
-    parser = ArgumentParser(description='Tool for removing files generated by pyccel')
+    Add the pyccel-clean arguments to the parser for command line arguments.
 
+    Parameters
+    ----------
+    parser : argparse.ArgumentParser
+        The parser to be modified.
+    """
     parser.add_argument('folders', metavar='N', type=str, nargs='*',
             help='The folders to be cleaned (default is the current folder')
-    parser.add_argument('-n', '--not-recursive', action='store_false',
+    parser.add_argument('-n', '--not-recursive', action='store_false', dest='recursive',
             help='Only run pyccel-clean in the current directory. Do not recurse into other folders')
-    parser.add_argument('-s', '--remove-libs', action='store_true',
+    parser.add_argument('-s', '--remove-libs', action='store_true', dest='remove_shared_libs',
             help='Also remove any libraries generated by Python from the folder. Beware this may remove shared libraries generated by tools other than pyccel')
     parser.add_argument('-p', '--remove-programs', action='store_true',
             help='Also remove any programs from the folder. Beware this may remove programs unrelated to pyccel')
+
+def pyccel_clean_command():
+    """
+    Command line wrapper around the pyccel_clean function.
+
+    A wrapper around the pyccel_clean function which allows
+    command line arguments to be passed to the function.
+    """
+    parser = ArgumentParser(description='Tool for removing files generated by pyccel', add_help=True)
+
+    #... Help and Version
+    add_version_flag(parser)
+
+    setup_pyccel_clean_parser(parser)
+
     args = parser.parse_args()
 
-    folders = args.folders
-    recursive = args.not_recursive
-    remove_libs = args.remove_libs
-    remove_programs = args.remove_programs
+    print("warning: The pyccel-clean command is deprecated and will be removed in v2.3. Please use pyccel clean.", file=sys.stderr)
 
+    pyccel_clean_loop(**vars(args))
+
+    Call the `pyccel_clean_single_folder` function in a loop over multiple folders.
+
+    Call the `pyccel_clean_single_folder` function in a loop over multiple folders.
```

Comment:
Sorry, my suggestion was missing two lines of code (unchanged) for some reason

```suggestion
def pyccel_clean_loop(folders, **kwargs):
    """
    Call the `pyccel_clean_single_folder` function in a loop over multiple folders.

    Call the `pyccel_clean_single_folder` function in a loop over multiple folders.
```